pkg/selinux/selinux-client.js

```diff
@@ -108,8 +110,18 @@ export function init(statusChangedCallback) {
         // Example:
         // authlogin_nsswitch_use_ldap (on , on) Allow authlogin to nsswitch use ldap
         // Split by first ')', as the name cannot contain ')'
-        if (item)
-            result.push(item.split(")", 2)[1].trim());
+        if (item) {
+            const description = item.split(")", 2)[1].trim();
+            const state = item.split("(", 2)[1].startsWith("on") ? "yes" : "no";
+            const name = item.split(" ")[0];
+            const ansible = `
+- name: ${description}
+  seboolean:
+    name: ${name}
+    state: ${state}
+    persistent: yes\n`;
```

Comment:
nitpick: seems a bit odd to mix `\n` and actual line breaks in the same string. Just put the ` on the next line?